app/src/main/java/com/asfoundation/wallet/promotions/PromotionsFragment.kt

```diff
@@ -101,14 +122,14 @@ class PromotionsFragment : BasePageViewFragment(), PromotionsView {
     no_network.visibility = GONE
     retry_animation.visibility = GONE
     no_promotions.visibility = VISIBLE
-    locked_promotions.visibility = GONE
+    locked_promotions_no_vouchers.visibility = GONE
   }
 
   override fun showLockedPromotionsScreen() {
     no_network.visibility = GONE
     retry_animation.visibility = GONE
     no_promotions.visibility = GONE
-    locked_promotions.visibility = VISIBLE
+    locked_promotions_no_vouchers.visibility = VISIBLE
```

Comment:
We should also hide the content related to vouchers for unknown users. Considering this (unlikely but possible) use case:

User is unknown and had vouchers - has wallet on bg with promotions screen. The vouchers have, however, been disabled since he returned. So the view would have to go from locked with vouchers to locked without.

The same use case could also apply to `showNoPromotionsScreen()` and `showNetworkErrorView()`